driver/evrmemmap.h

```diff
@@ -97,12 +97,19 @@ struct DBufInfo {
 /* These must be powers of two!!! */
 #define MAX_EVR_EVTQ 1024
 #define MAX_EVR_DBQ     4
+#define MAX_EVR_DBQ2   16
```

Comment:
Is there a reason for 16 specifically or was it a "`4*MAX_EVR_DBQ` should be good enough for the issue at hand"?

Reply:
I saw cases when more than 4 fiducial events were processed in a single call of the handler, but never more than 8.  So I moved this to 16 out of an abundance of caution.

Reply:
Gotcha - that may be worth throwing in as a comment, perhaps